etc/asterisk/ari.d/01-wazo.conf

```diff
@@ -1,7 +1,7 @@
 [general]
 enabled = yes
 allowed_origins = *
-channelvars = CHANNEL(linkedid),CHANNEL(videonativeformat),WAZO_ANSWER_TIME,WAZO_CALL_RECORD_ACTIVE,WAZO_CALL_RECORD_SIDE,WAZO_CHANNEL_DIRECTION,WAZO_DEREFERENCED_USERUUID,WAZO_ENTRY_CONTEXT,WAZO_ENTRY_EXTEN,WAZO_LINE_ID,WAZO_LOCAL_CHAN_MATCH_UUID,WAZO_SIP_CALL_ID,WAZO_SWITCHBOARD_QUEUE,WAZO_SWITCHBOARD_HOLD,WAZO_TENANT_UUID,XIVO_BASE_EXTEN,XIVO_ON_HOLD,XIVO_USERUUID
+channelvars = CHANNEL(linkedid),CHANNEL(videonativeformat),WAZO_ANSWER_TIME,WAZO_CALL_DIRECTION,WAZO_CALL_RECORD_ACTIVE,WAZO_CALL_RECORD_SIDE,WAZO_CHANNEL_DIRECTION,WAZO_CONVERSATION_DIRECTION,WAZO_DEREFERENCED_USERUUID,WAZO_ENTRY_CONTEXT,WAZO_ENTRY_EXTEN,WAZO_LINE_ID,WAZO_LOCAL_CHAN_MATCH_UUID,WAZO_SIP_CALL_ID,WAZO_SWITCHBOARD_QUEUE,WAZO_SWITCHBOARD_HOLD,WAZO_TENANT_UUID,XIVO_BASE_EXTEN,XIVO_ON_HOLD,XIVO_USERUUID
```

Comment:
AFAIU, `WAZO_CALL_DIRECTION` is not needed in the events. Is this correct?
FYI we should limit the `channelvars` as much as we can, because it adds overhead to *all* AMI / ARI messages. I guess we will eventually have to make Asterisk send `channelvars` only on certain events.

Reply:
Oh indeed it is not needed in the events, it is computed. I will remove it and only leave the conversation direction. Thank you!